.github/workflows/deployment.yml

```diff
@@ -0,0 +1,52 @@
+name: OpenEduHub - Github Page
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    name: Deploy to GitHub Pages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ./repo
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: ./repo
+          file: ./repo/Dockerfile
+          push: false
+          load: true
+          tags: ccas/docusaurus:latest
+          cache-from: type=gha
+          cache-to: type=gha
+
+      - name: Load image
+        run: |
+          mkdir output
+          docker image list
+          docker run -v $GITHUB_WORKSPACE/repo:/content -v $GITHUB_WORKSPACE/output:/output ccas/docusaurus:latest
+
+      # Popular action to deploy to GitHub Pages:
+      # Docs: https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-docusaurus
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          personal_token: ${{ secrets.ACCESS_TOKEN }}
```

Comment:
@razvand Did you add the ACCESS_TOKEN to the project?

Reply:
@Sergiu121 , no, I didn't. Where should I add that?

Reply:
This would only need to be added to the [final repository](https://github.com/open-education-hub/ccas), right?

Reply:
Exactly. You will have to add it as a secret as shown in [this tutorial](https://www.youtube.com/watch?v=IuT0Ua7V4xA).